### ISSUE TITLE
discoverDomain.sh fails due to __handle_unexpected_exception

### ISSUE THREAD

**Issue by dbeniteza** [oracle/weblogic-deploy-tooling#1212]
Running WDT 2.3.3 discoverDomain tool fails with the following exception:
```
$ /admin/utils/weblogic-deploy/bin/discoverDomain.sh -oracle_home /software/WL_10_3_6 \
>  -domain_home /domains/wld-pre-32 \
>  -skip_archive  \
>  -model_file /export/home/bea/test_dani/model_32.yaml \
>  -admin_user weblogic \
>  -admin_url t3://wls-pre-32-sa-adm-n0:13200
JDK version is 1.8.0_251-b08
JAVA_HOME = /admin/jdk/jdk8
WLST_EXT_CLASSPATH = /admin/utils/weblogic-deploy/lib/weblogic-deploy-core.jar
CLASSPATH = /admin/utils/weblogic-deploy/lib/weblogic-deploy-core.jar
WLST_PROPERTIES = -Dcom.oracle.cie.script.throwException=true -Djava.util.logging.config.class=oracle.weblogic.deploy.logging.WLSDeployLoggingConfig
/software/WL_10_3_6/oracle_common/common/bin/wlst.sh /admin/utils/weblogic-deploy/lib/python/discover.py -oracle_home /software/WL_10_3_6 -domain_home /domains/wld-pre-32 -skip_archive -model_file /export/home/bea/test_dani/model_32.yaml -admin_user weblogic -admin_url t3://wls-pre-32-sa-adm-n0:13200

CLASSPATH=/software/WL_10_3_6/patch_wls1036/profiles/default/sys_manifest_classpath/weblogic_patch.jar:/software/WL_10_3_6/patch_ocp371/profiles/default/sys_manifest_classpath/weblogic_patch.jar:/admin/jdk/jdk8/lib/tools.jar:/software/WL_10_3_6/wlserver_10.3/server/lib/weblogic_sp.jar:/software/WL_10_3_6/wlserver_10.3/server/lib/weblogic.jar:/software/WL_10_3_6/modules/features/weblogic.server.modules_10.3.6.0.jar:/software/WL_10_3_6/wlserver_10.3/server/lib/webservices.jar:/software/WL_10_3_6/modules/org.apache.ant_1.7.1/lib/ant-all.jar:/software/WL_10_3_6/modules/net.sf.antcontrib_1.1.0.0_1-0b2/lib/ant-contrib.jar:/admin/utils/weblogic-deploy/lib/weblogic-deploy-core.jar:/software/WL_10_3_6/oracle_common/modules/oracle.jrf_11.1.1/jrf-wlstman.jar:/software/WL_10_3_6/oracle_common/common/wlst/lib/adf-share-mbeans-wlst.jar:/software/WL_10_3_6/oracle_common/common/wlst/lib/adfscripting.jar:/software/WL_10_3_6/oracle_common/common/wlst/lib/mdswlst.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/auditwlst.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/igfwlsthelp.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/jps-wls-trustprovider.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/jps-wlst.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/jrf-wlst.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/oamAuthnProvider.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/oamap_help.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/ossoiap.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/ossoiap_help.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/ovdwlsthelp.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/sslconfigwlst.jar:/software/WL_10_3_6/oracle_common/common/wlst/resources/wsm-wlst.jar:/software/WL_10_3_6/utils/config/10.3/config-launch.jar::/software/WL_10_3_6/wlserver_10.3/common/derby/lib/derbynet.jar:/software/WL_10_3_6/wlserver_10.3/common/derby/lib/derbyclient.jar:/software/WL_10_3_6/wlserver_10.3/common/derby/lib/derbytools.jar::
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=128m; support was removed in 8.0

Initializing WebLogic Scripting Tool (WLST) ...

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

####<Oct 5, 2022 1:08:47 PM> <INFO> <WebLogicDeployToolingVersion> <logVersionInfo> <WLSDPLY-01750> <The WebLogic Deploy Tooling discoverDomain version is 2.3.3:.3ad6d61:Aug 26, 2022 19:01 UTC>
####<Oct 5, 2022 1:08:47 PM> <INFO> <WLSDeployLoggingConfig> <logLoggingDirectory> <WLSDPLY-01755> <The discoverDomain program will write its log to directory /admin/utils/weblogic-deploy/logs>
Please enter the WebLogic administrator password:
####<Oct 5, 2022 1:08:54 PM> <INFO> <ModelContext> <__copy_from_args> <WLSDPLY-01050> <WebLogic version for aliases is 10.3.6.0>
####<Oct 5, 2022 1:08:54 PM> <INFO> <discover> <main> <WLSDPLY-06024> <No variable file provided. Model passwords will contain the token '-- FIX ME --'6>

####<Oct 5, 2022 1:08:58 PM> <INFO> <discover> <_get_domain_name> <WLSDPLY-06022> <Discover domain wld-pre-32>
Problem invoking WLST - Traceback (innermost last):
  File "/admin/utils/weblogic-deploy/lib/python/discover.py", line 641, in ?
AttributeError: 'module' object has no attribute '__handle_unexpected_exception'


discoverDomain.sh completed but with some issues (exit code = 1)
```

See attached discoverDomain.log file.
It seems to be something related to 2.3.3 version for WLS 11g. I've run the discoverDomain tool 2.1.0 version with no issues at all (same WLS and JDK version)
The problem doesn't occur for WLS 12g.

$ cat VERSION.txt
WebLogic Deploy Tooling 2.3.3

[discoverDomain.log](https://github.com/oracle/weblogic-deploy-tooling/files/9715083/discoverDomain.log)

**Comment by robertpatrick** [comment]
@dbeniteza This is already fix in the main branch.

**Comment by robertpatrick** [comment]
Fixed by #1208
